config/main.py

```diff
@@ -744,11 +749,17 @@ def _restart_services(config_db):
     if asic_type == 'mellanox' and 'pmon' in services_to_restart:
         services_to_restart.remove('pmon')
 
-    execute_systemctl(services_to_restart, SYSTEMCTL_ACTION_RESTART)
+    try:
+        execute_systemctl(services_to_restart, SYSTEMCTL_ACTION_RESTART)
 
-    # Reload Monit configuration to pick up new hostname in case it changed
-    click.echo("Reloading Monit configuration ...")
-    clicommon.run_command("sudo monit reload")
+        # Reload Monit configuration to pick up new hostname in case it changed
+        click.echo("Reloading Monit configuration ...")
+        clicommon.run_command("sudo monit reload")
+    except Exception as e:
+        log.log_error("'_restart_verify_services' start services failed, error: {}".format(e))
```

Comment:
This log message is strange. `_restart_verify_services` is not the name of the function. Probably should read:

```python3
log.log_error("'_restart_services' failed, error: {}".format(e))
```

Reply:
Modified.